packages/server/graphql/types/AtlassianIntegration.ts

```diff
@@ -157,17 +158,9 @@ const AtlassianIntegration = new GraphQLObjectType<any, GQLContext>({
       resolve: async (
         {accessToken, cloudIds, teamId, userId}: AtlassianAuth,
         _args: unknown,
-        {authToken}: GQLContext
+        context
       ) => {
-        const viewerId = getUserId(authToken)
-        if (viewerId !== userId) return []
```

Comment:
-1 is this safe to remove? if you're a consultant in multiple jira clouds & i'm on your team in parabol, I could call

```gql
viewer {
  teamMember(teamId: 'mattAndNick', userId: 'nickO') {
    integrations {
      atlassian {
        projects {...}
      }
    }
  }
}
```

and get access to all your projects using your access token

Reply:
Doh sorry, that wasn't deliberate! While refactoring and removing the code below, I accidentally removed those lines too